sql/core/src/main/scala/org/apache/spark/sql/expressions/Aggregator.scala

```diff
@@ -49,6 +49,7 @@ import org.apache.spark.sql.execution.aggregate.TypedAggregateExpression
  * @tparam OUT The type of the final output result.
  * @since 1.6.0
  */
+@SerialVersionUID(2093413866369130093L)
```

Comment:
Why is this needed?

Reply:
`TypedColumn`?

Reply:
It's for mapping the client's `Aggregator` class to this one. This is required because we now serialise the whole `Aggregator` instance on the client side. If we decide to serialise individual methods instead, this `SerialVersionUID` can be abolished. Wdyt?

Reply:
I tried multiple methods and still can't succeed without this UID. As long as we want users to use UDAF like this:
```
new Aggregator {
   def merge(...) = {...}
   ...
}
```
The serialized payload (either the whole Aggregator instance or individual methods) will carry a reference to the Aggregator instance that needs to be decoded on the server side. Without this UID, it will throw an exception.